compiler/rustc_mir/src/util/borrowck_errors.rs

```diff
@@ -68,8 +68,8 @@ impl<'cx, 'tcx> crate::borrow_check::MirBorrowckCtxt<'cx, 'tcx> {
             err.span_label(
                 new_loan_span,
                 format!(
-                    "mutable borrow starts here in previous \
-                     iteration of loop{}",
+                    "mutable already borrowed in the previous \
+                     iteration of the loop{}",
                     opt_via
```

Comment:
You probably want to use `desc` here:
```suggestion
                    "{} was borrowed mutably in the previous iteration of the loop{}",
                    desc,
                    opt_via,
```

Reply:
@PankajChaudhary5 Can you please address this comment?

Reply:
@camelid they did, they just didn't hit the 'resolve' button.

Reply:
@jyn514 They did not address changing the message. I find the current message quite confusing:

> `mutable {}{} already borrowed in the previous iteration of the loop{}`

which renders as:

> `mutable `x` already borrowed in the previous iteration of the loop`